API.md

```diff
@@ -316,6 +319,8 @@ Reloads the player with the given configuration. If the configuration object is
 | watermark | <code>object</code> | Watermark image URL. |
 | withCredentials | <code>boolean</code> | Indicates whether or not cross-site Access-Control requests should be made using credentials such as cookies, authorization headers or TLS client certificates. |
 | cmcd | <code>Object</code> |  Enables the Common Media Client Data. Only needs to create an empty object to enable it. |
+|licenseWithCredentials| <code>boolean</code> | Indicates whether or not cross-site Access-Control requests should be made using credentials such as cookies, authorization headers or TLS client certificates with DRM.|
+| maxFrameDrop | <code>number</code> | The value of this property should be within the range 0 - 1.We have added a new event that is triggered whenever the percentage of frames dropped is higher than the threshold established in the nexplayer.|
```

Comment:
Remove this line: "We have added a new event that is triggered whenever the percentage of frames dropped is higher than the threshold established in the nexplayer."

Instead write: "A Frames_Drop_Capping event will be triggered whenever the percentage of frames dropped is higher than this threshold". Also link the event with its corresponding section in the documentation.